static/styles/zulip.css

```diff
@@ -2877,107 +2911,109 @@ select.inline_select_topic_edit {
 }
 
 @media (width < $sm_min) {
-    .column-right.expanded .right-sidebar,
-    .column-left.expanded .left-sidebar {
-        margin-top: 31px;
-    }
+    body {
```

Comment:
Added all these styles under `body` so that these properties can override values set by spectator view without using important.